frontend/src/components/auth/Signup.tsx

```diff
@@ -1,7 +1,9 @@
 import React, { useContext, useState } from "react";
-import { Redirect } from "react-router-dom";
+import { Redirect, useLocation } from "react-router-dom";
 
 import authAPIClient from "../../APIClients/AuthAPIClient";
+// needs update when welcome page is available
+// import { HOME_PAGE , WELCOME_PAGE} from "../../constants/Routes";
```

Comment:
yeee if you merge in latest `main` you'll get Jeffery's welcome page stuff, so the `WELCOME_PAGE` route constant will be available for you to use:
`<Redirect to={WELCOME_PAGE} />`
instead of
`<Redirect to="/welcome" />`
in line 25 below